packages/textfield/index.js

```diff
@@ -1,12 +1,13 @@
 import { css, html } from 'lit';
 
-import { input, label as l, helpText as h } from '@warp-ds/css/component-classes';
+import { classNames } from '@chbphone55/classnames';
+import { input as ccInput, label as ccLabel, helpText as ccHelpText } from '@warp-ds/css/component-classes';
 import WarpElement from '@warp-ds/elements-core';
 import { ifDefined } from 'lit/directives/if-defined.js';
 
-import { classes } from '../utils';
+import { kebabCaseAttributes } from '../utils';
 
-class WarpTextField extends WarpElement {
+class WarpTextField extends kebabCaseAttributes(WarpElement) {
```

Comment:
Same here with the defined `attribute: 'some-attribute-name'` options in `static properties`. We can remove those in favour of the wrapper doing the same thing.

Reply:
Updated in this commit: [5459cf7](https://github.com/warp-ds/elements/pull/174/commits/5459cf7eeceee45bb633efda91311217978ae8df)